joomla/joomla-cms
# Add  vertical ordering to blog layouts

Pull Request for Issue # .

### Summary of Changes
Proof of Concept 
There are many complaints of users who don't know how to add modifier classes for Blog Layouts. 
This PR is a proof of concept where the main classes for the modifiers are added to the blog layout params. 
I would like to have some feedback and if it is accepted can add it to featured. 

Open questions are, for example:
- Is it self explaining and helpful? 
- Do we need global params for that ?

### Testing Instructions
You need several articles in a category.

Now make a menu item for a category blog. In the Tab Blog Layout There are two new selections: Default classes. 

![grafik](https://user-images.githubusercontent.com/1035262/95001857-60c72d80-05ce-11eb-8afb-c2b515905954.png)
One for leading articles (the screenshot), the same for intro articles.

If the user selects "yes", the shown presents new input fields: 

![grafik](https://user-images.githubusercontent.com/1035262/95001927-24e09800-05cf-11eb-84e5-7ec67deecfa6.png)
The same for intro articles

Select here what you want, play with different combinations and see what happens with your blog. For example this: 

![grafik](https://user-images.githubusercontent.com/1035262/95001976-96b8e180-05cf-11eb-898f-5e1cbf9be5f3.png)


### Actual result BEFORE applying this Pull Request
There is one input field for the classes of the leading articles and one for the intro articles. 


### Expected result AFTER applying this Pull Request
There is one input field for the classes of the leading articles and one for the intro articles. 
There are input fields for default classes.

### Documentation Changes Required
yes
